core/src/test/java/org/infinispan/stream/BaseStreamTest.java

```diff
@@ -1489,6 +1489,48 @@ public void testDoubleAllMatch() {
       assertFalse(createStream(entrySet).mapToDouble(toDouble).allMatch(i -> Math.floor(i) == i));
    }
 
+   public void testSkip() {
+      Cache<Double, String> cache = getCache(0);
+      int range = 10;
+      // First populate the cache with a bunch of values
+      IntStream.range(0,range).mapToDouble(value -> value).forEach(i -> cache.put(i, i + "-value"));
+      CacheSet<Map.Entry<Double, String>> entrySet = cache.entrySet();
+
+      int cnt = range;
+      for(int i = 0;i<cnt;i++){
```

Comment:
Formatting is off, please apply the formatter found at https://github.com/infinispan/infinispan/tree/master/ide-settings